taxi/models.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
```

Comment:
According to the checklist, it is recommended to define all attributes in the model before defining the `Meta` class. Although it's not present here, keep this in mind for future models.